docs/entities.rst

```diff
@@ -45,7 +44,7 @@ Custom, Domain-specific entities
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In the introductory tutorial we build a restaurant bot, and create custom entities for location and cuisine.
-The best components for training these domain-specific entity recognisers are the ``ner_mitie`` and ``ner_crf`` components. 
+The best components for training these domain-specific entity recognisers is the ``ner_crf`` component.
 It is recommended that you experiment with both of these to see what works best for your data set. 
```

Comment:
no longer relevant

Reply:
why not? makes clear that you should use it instead of duckling for your domain specific entities

Reply:
I'm talking about this sentence "It is recommended that you experiment with both of these to see what works best for your data set."  😄

Reply:
ah yes, sure :D